recipes/libvpx/all/conanfile.py

```diff
@@ -135,7 +109,11 @@ def generate(self):
         if host_os == 'Windows':
             os_name = 'win32' if self.settings.arch == 'x86' else 'win64'
         elif is_apple_os(self):
-            if self.settings.arch in ["x86", "x86_64"]:
+            # Solves cross-building for iOS
+            # Issue related: https://github.com/conan-io/conan-center-index/issues/20513
+            if self.settings.os == "iOS":
+                os_name = 'iphonesimulator'
```

Comment:
Diff is unclear.

The only addition was these 4 lines. The rest is the same.

Reply:
I don't think I understand what this special condition means. iOS device can be armv7 or armv8, iOS simulator can be armv8 or x86_64 (let's not think about ancient x86 simulators). Why forcing simulator here?

Reply:
I'm forcing the name because of the libvpx configure logic. The target name is necessary for the configure script to choose the correct toolchain. The configure only has these related to iOS:

```
    . . .
    x86-iphonesimulator-gcc
    . . .
    x86_64-iphonesimulator-gcc
```
So what I have to check is the iOS SDK that you put in your Conan profile https://github.com/conan-io/conan-center-index/pull/22263/commits/3cba6e0464c265efd5761669cab6b5c99573e74c

Reply:
so you're saying that basically libvpx cannot be built for iOS device? That's quite weird...

I see there's armv7-darwin-gcc in the list - this looks exactly as 32-bit iOS device, and I believe arm64-darwin*-gcc would be 64-bit iOS device (or Apple SIlicon Mac).